src/settings-ui/Settings.UI/SettingsXAML/Views/MouseUtilsPage.xaml

```diff
@@ -289,7 +289,7 @@
                                 </ToolTipService.ToolTip>
                                 <Button.Flyout>
                                     <Flyout x:Uid="MouseJumpThumbnailSize_Edit">
-                                        <StackPanel Margin="0,12,0,0" Spacing="16">
+                                        <StackPanel Spacing="16">
```

Comment:
Why removing here and keeping on `ImageResizerPage.xaml`?

Reply:
No actual reason, just missed it. Thanks :)